packages/features/calendars/lib/getAvailableDatesInMonth.test.ts

```diff
@@ -35,5 +35,30 @@ describe("Test Suite: Date Picker", () => {
 
       expect(result).toHaveLength(1);
     });
+
+    test("it translates correctly regardless of system time", () => {
+      {
+        // test a date in negative UTC offset
+        vi.useFakeTimers().setSystemTime(new Date("2023-10-24T13:27:00.000-07:00"));
+
+        const currentDate = new Date();
+        const result = getAvailableDatesInMonth({
+          browsingDate: currentDate,
+        });
+
+        expect(result).toHaveLength(daysInMonth(currentDate) - currentDate.getDate() + 1);
+      }
+      {
+        // test a date in negative UTC offset
+        vi.useFakeTimers().setSystemTime(new Date("2023-10-24T13:27:00.000+07:00"));
```

Comment:
Since `currentDate` is `new Date()`, this won't start failing tomorrow/next month/etc. will it? 😅

Reply:
If you pass new Date() a full ISO string; it'll always stay equal to ""2023-10-24T13:27:00.000+07:00""